src/crimson/os/futurized_store.h

```diff
@@ -45,12 +48,39 @@ class FuturizedStore {
 
   using mount_ertr = crimson::errorator<crimson::stateful_ec>;
   virtual mount_ertr::future<> mount() = 0;
+
   virtual seastar::future<> umount() = 0;
 
   using mkfs_ertr = crimson::errorator<crimson::stateful_ec>;
   virtual mkfs_ertr::future<> mkfs(uuid_d new_osd_fsid) = 0;
+
   virtual seastar::future<store_statfs_t> stat() const = 0;
 
+  virtual uuid_d get_fsid() const  = 0;
+
+  virtual seastar::future<> write_meta(const std::string& key,
+				       const std::string& value) = 0;
+
+  virtual FuturizedShardStore& get_sharded_store() = 0;
```

Comment:
Hmm, seems like this should probably take a core_id_t?

Reply:
Or does it have to be called on the shard on which it will be used?  Please add a comment in the standard interface style explaining the rules.